packages/core/src/tracing/spanstatus.ts

```diff
@@ -10,6 +10,7 @@ export const SPAN_STATUS_ERROR = 2;
  * @param httpStatus The HTTP response status code.
  * @returns The span status or unknown_error.
  */
+// https://develop.sentry.dev/sdk/event-payloads/span/
```

Comment:
l (feel free to ignore): Any reason for not adding this link directly to the JSDoc header?

Reply:
Ok fair point, in this case I think it's totally fine. In other cases I think it's still worth to take that risk but admittedly, I didn't think too much about it until now 😅